### ISSUE TITLE
[Accessibility][Usable - Property Pane_Ask a question (Bot Asks)]: Focus goes on 'Here' present into the error message without expanding the error message.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2114]
**User Experience** 
Low vision users who rely on keyboard will face difficulty and easily confused if the focus moves to the link in error message without expanding whole error message.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen.
5. Now, navigate to 'Prompt' edit field and delete all the content & press space.
6. Now, navigate forward with tab key and Observe the issue.

**Actual:** 
On entering blank data in the prompt edit field, an error message appeared with 'Here' as link which gets the focus with Tab key but the whole error message is not expanding. Due to which it looks like the focus goes to the hidden element.

**Expected:**
As soon as the interactive element gets the keyboard focus the error message should be expended.

**Note:**
Same issue is observed for all the Property Pane which have prompt or any syntax edit fields appeared at right side of the screen.

**MAG Reference:** [MAG](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BD0F24E02-1D9C-4DA2-8F5B-2A3C1176F3BF%7D&file=Guideline%2017%20A%20-%20Contrast%20(Minimum)%20When%20Conveying%20Information.docx&action=edit&mobileredirect=true)

**Attachment:**
[Usable_focus goes into the error message without expanding.zip](https://github.com/microsoft/BotFramework-Composer/files/4261750/Usable_focus.goes.into.the.error.message.without.expanding.zip)
![focus goes into the error without expanding them](https://user-images.githubusercontent.com/60913131/75662019-69e8c880-5c94-11ea-87d4-fe56504c2d93.png)

**Comment by beyackle** [comment]
There's still a problem here, but not as the issue describes. Following all the steps does create a red error message with the "click-to-expand" arrows in it, but hitting Tab at that point just inserts a tab into the editor instead of moving to the next element. I can even start the focus somewhere earlier in the page and use tab to navigate into the editor, but as soon as I'm there, I'm stuck.

**Comment by beyackle** [comment]
Okay, apparently there's a documented way to break out of a Monaco editor using keyboard nav (Ctrl-M or Ctrl-Shift-M on Mac, as documented in https://github.com/Microsoft/monaco-editor/wiki/Monaco-Editor-Accessibility-Guide). So this isn't as big a problem as I thought if KO users are aware of that.

**Comment by beyackle** [comment]
Now that I've messed with this a bit, the problem is very much as the original issue states. After causing an error and hitting Ctrl-M to restore the Tab key's keyboard-nav function, a single tab will take me to this weird non-space. A second tab then moves onto the reveal-more arrow.

![image](https://user-images.githubusercontent.com/61990921/76986646-28e0ed00-68ff-11ea-8275-100ca5e2fd36.png)

**Comment by beyackle** [comment]
Possible solution: just make the error message multiline without the "reveal" arrows (also, as per the MessageBar examples in Fabric, use a real Link component instead of creating our own with an <a> tag). I can't vouch for the syntax of this message - I found where it's coming from, and that's over in SDK-land - but the mechanism works fine.

![image](https://user-images.githubusercontent.com/61990921/77005345-e084f780-691d-11ea-90f4-ff8140c99361.png)

**Comment by beyackle** [comment]
I spoke to @sangwoohaan about this and this looks like a good solution as long as the error messages don't get too long (which, from what I've seen of them, they don't).

**Comment by beyackle** [comment]
Note: this is blocked temporarily until #2118 is merged; once it does I'll have to move the changes over to the newly relocated code.

**Comment by beyackle** [comment]
Looks like the new editor is out and I've got it all building and happy, so I'm going to port my fix over now.

**Comment by ashish315** [comment]
@corinagum As checked the issue is fixed as the error message is short now. So, basically the opened issue is fixed. So, HCL is closing the bug.
But in order to fix the issue, now the error message is not clearly visible as last word of the error message gets trimmed and their is no way to expand the error message. This issue creates problem for normal user. So, now its upto you fix this new issue.
PFSS
![image](https://user-images.githubusercontent.com/60913131/79757500-ccc41b00-8339-11ea-8cea-8eca37eceb34.png)
